Cryptomator.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -19,6 +19,15 @@
           "version": "1.2.4"
         }
       },
+      {
+        "package": "AWSiOSSDKV2",
+        "repositoryURL": "https://github.com/aws-amplify/aws-sdk-ios-spm",
+        "state": {
+          "branch": null,
+          "revision": "443e35abb0646fb685e0754db74249d3242625ab",
+          "version": "2.27.10"
+        }
+      },
```

Comment:
I suppose this should be added in a different PR, right?

Reply:
This comes automatically through the update of cloud-access-swift, which already has the dependency to `AWSiOSSDKV2`. Therefore, as already discussed internally, this can be considered resolved.